### ISSUE TITLE
I run the train.py and got this,I think there is something wrong,and I do not know why.

### ISSUE THREAD

**Issue by Kaido0** [jocicmarko/ultrasound-nerve-segmentation#24]
I use 18 images to train the model, and I got loss like this~I do not know how to solve this?Is the training sets too small?

Epoch 2/20
19/19 [==============================] - 3s - loss: -0.0191 - dice_coef: 0.0191
Epoch 3/20
19/19 [==============================] - 3s - loss: -0.0191 - dice_coef: 0.0191
Epoch 4/20
19/19 [==============================] - 4s - loss: -0.0191 - dice_coef: 0.0191
Epoch 5/20
19/19 [==============================] - 4s - loss: -0.0191 - dice_coef: 0.0191
Epoch 6/20
19/19 [==============================] - 4s - loss: -0.0191 - dice_coef: 0.0191
Epoch 7/20
19/19 [==============================] - 3s - loss: -0.0191 - dice_coef: 0.0191
Epoch 8/20
19/19 [==============================] - 4s - loss: -0.0191 - dice_coef: 0.0191
Epoch 9/20
19/19 [==============================] - 4s - loss: -0.0191 - dice_coef: 0.0191
Epoch 10/20
19/19 [==============================] - 3s - loss: -0.0192 - dice_coef: 0.0192
Epoch 11/20
19/19 [==============================] - 3s - loss: -0.0192 - dice_coef: 0.0192
Epoch 12/20
19/19 [==============================] - 3s - loss: -0.0192 - dice_coef: 0.0192
Epoch 13/20
19/19 [==============================] - 3s - loss: -0.0192 - dice_coef: 0.0192
Epoch 14/20
19/19 [==============================] - 3s - loss: -0.0192 - dice_coef: 0.0192
Epoch 15/20
19/19 [==============================] - 3s - loss: -0.0192 - dice_coef: 0.0192
Epoch 16/20
19/19 [==============================] - 3s - loss: -0.0192 - dice_coef: 0.0192
Epoch 17/20
19/19 [==============================] - 4s - loss: -0.0193 - dice_coef: 0.0193
Epoch 18/20
19/19 [==============================] - 4s - loss: -0.0193 - dice_coef: 0.0193
Epoch 19/20
19/19 [==============================] - 3s - loss: -0.0193 - dice_coef: 0.0193
Epoch 20/20
19/19 [==============================] - 4s - loss: -0.0193 - dice_coef: 0.0193

**Comment by jocicmarko** [comment]
18 images is WAY too low to train a deep neural network. You won't be able to train it and/or you will overfit almost certainly. You could try:
- getting more data - depends on the problem, but I'd say at least 1k images
- using pretrained models - but even with pretrained models 18 images is too few
